diff --git a/node_modules/@nuxt/vue-app/template/App.js b/node_modules/@nuxt/vue-app/template/App.js
index c8f7c95..9c65c53 100644
--- a/node_modules/@nuxt/vue-app/template/App.js
+++ b/node_modules/@nuxt/vue-app/template/App.js
@@ -43,9 +43,6 @@ export default {
     <% if (features.layouts) { %>
     const layoutEl = h(this.layout || 'nuxt')
     const templateEl = h('div', {
-      domProps: {
-        id: '__layout'
-      },
       key: this.layoutName
     }, [layoutEl])
     <% } else { %>
@@ -69,11 +66,8 @@ export default {
     }, [templateEl])
     <% } %>
 
-    return h('div', {
-      domProps: {
-        id: '<%= globals.id %>'
-      }
-    }, [
+    return h('div', {}, 
+    [
       <% if (loading) { %>loadingEl, <% } %>
       <% if (buildIndicator) { %>h(NuxtBuildIndicator), <% } %>
       <% if (features.transitions) { %>transitionEl<% } else { %>templateEl<% } %>
