diff --git a/node_modules/github-slugger/index.js b/node_modules/github-slugger/index.js
index 9d40bc2..ffe4026 100644
--- a/node_modules/github-slugger/index.js
+++ b/node_modules/github-slugger/index.js
@@ -1,10 +1,20 @@
+// Modified with patch-package
+
 var emoji = require('emoji-regex')
 
 module.exports = BananaSlug
 
 var own = Object.hasOwnProperty
 var whitespace = /\s/g
-var specials = /[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&()*+,./:;<=>?@[\]^`{|}~’]/g
+var specials = /['!"#$%&()*+,/:;=?@[\]^`{|}~]/g
+var periodAtEnd = /\.{1}$/g
+// var shouldBecomeDash = /(?<!\.)\.{1,2}(?!\.)/g
+var shouldBecomeDash = /\./g
+var icons = /<[Ii]con.[^/]+\/*>(<\/[Ii]con>)*/
+var tagsToRemove = /<(?:E2EOnlyBadge|ComponentOnlyBadge)\s*\/?>/g
+var lessThan = /</g
+var greaterThan = />/g
+var twoOrMoreDashes = /-{2,}/g
 
 function BananaSlug () {
   var self = this
@@ -45,12 +55,32 @@ BananaSlug.prototype.reset = function () {
 
 function slugger (string, maintainCase) {
   if (typeof string !== 'string') return ''
-  if (!maintainCase) string = string.toLowerCase()
+  // We're removing this because we want to maintain casing for anchors 
+  // to be reverse-compatible with older docs
+  // if (!maintainCase) string = string.toLowerCase()
 
-  return string.trim()
-    .replace(specials, '')
+  var str = string.trim()
+    .replace(icons, '')
+    .replace(tagsToRemove, '')
+    .trim()
+    .replace(periodAtEnd, '')
+    .replace(shouldBecomeDash, '-')
+    .replace(lessThan, 'lt-')
+    .replace(greaterThan, '-gt')
+    .replace(specials, '-')
     .replace(emoji(), '')
     .replace(whitespace, '-')
+    .replace(twoOrMoreDashes, '-')
+
+  if (str[str.length - 1] === '-') {
+    str = str.slice(0, str.length - 1)
+  }
+
+  // Used for debugging output
+  // console.log(`SLUG\t${str}\t${string}`)
+
+  return str
 }
 
 BananaSlug.slug = slugger
+
