diff --git a/node_modules/github-slugger/index.js b/node_modules/github-slugger/index.js
index 9d40bc2..6e69c3a 100644
--- a/node_modules/github-slugger/index.js
+++ b/node_modules/github-slugger/index.js
@@ -5,6 +5,7 @@ module.exports = BananaSlug
 var own = Object.hasOwnProperty
 var whitespace = /\s/g
 var specials = /[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&()*+,./:;<=>?@[\]^`{|}~’]/g
+var icons = /<[Ii]con.[^/]+\/*>(<\/[Ii]con>)*/g
 
 function BananaSlug () {
   var self = this
@@ -48,6 +49,8 @@ function slugger (string, maintainCase) {
   if (!maintainCase) string = string.toLowerCase()
 
   return string.trim()
+    .replace(icons, '')
+    .trim()
     .replace(specials, '')
     .replace(emoji(), '')
     .replace(whitespace, '-')
